arm-powerbiembedded/2016-01-29/swagger/powerbiembedded.json

```diff
@@ -0,0 +1,936 @@
+          "type": "string"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku"
+        },
+        "properties": {
+          "description": "Properties",
+          "type": "object"
+        }
+      }
+    },
+    "CreateWorkspaceCollectionRequest": {
+      "type": "object",
+      "properties": {
+        "location": {
+          "type": "string",
+          "description": "Azure location"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku"
+        }
+      }
+    },
+    "UpdateWorkspaceCollectionRequest": {
+      "type": "object",
+      "properties": {
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          }
+        },
+        "sku": {
+          "$ref": "#/definitions/AzureSku"
+        }
+      }
+    },
+    "CheckNameRequest": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Workspace collection name"
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type",
+          "default": "Microsoft.PowerBI/workspaceCollections"
+        }
+      }
+    },
+    "CheckNameResponse": {
+      "type": "object",
+      "properties": {
+        "nameAvailable": {
+          "type": "boolean",
+          "description": "Specifies a Boolean value that indicates whether the specified Power BI Workspace Collection name is available to use."
+        },
+        "reason": {
+          "type": "string",
+          "description": "Reason why the workspace collection name cannot be used.",
+          "enum": [
+            "Unavailable",
+            "Invalid"
+          ],
+          "x-ms-enum": {
+            "name": "CheckNameReason",
+            "modelAsString": true
+          }
+        },
+        "message": {
+          "type": "string",
+          "description": "Message indicating an unavailable name due to a conflict or a description of the naming rules that are violated."
+        }
+      }
+    },
+    "MigrateWorkspaceCollectionRequest": {
+      "type": "object",
+      "properties": {
+        "targetResourceGroup": {
+          "type": "string",
+          "description": "Name of the resource group that the Power BI Workspace Collections will be migrated to."
+        },
+        "resources": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          }
+        }
+      }
+    },
+    "Operation": {
+      "type": "object",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Operation name"
+        },
+        "display": {
```

Comment:
I think AutoRest would name it as DisplayModel and create a model class for you anyway. However, it would be nice if you can create a separate class may be named Display or something else and then reference it here.